static/locales/en-US.yaml

```diff
@@ -1040,6 +1041,7 @@ Description:
 #Tooltips
 Tooltips:
   General Settings:
+    Enable VIM key-bindings: Allow navigation using VIM hotkeys.
```

Comment:
You're not saying anything new here as the toggle says the same https://github.com/FreeTubeApp/FreeTube/pull/7336/files#diff-4eaadcc25bfbd7ab1ba912e36d5e1bcbab346cab59546a970df7546d5fb7a145R277

Remove all code related to the tooltip

Reply:
you removed the tooltip but not the translation